project/scalatest.scala

```diff
@@ -199,6 +201,11 @@ object ScalatestBuild extends Build {
       "org.scala-js" %% "scalajs-test-interface" % "0.6.17"
     )
 
+  def scalatestNativeLibraryDependencies =
+    Seq(
+      "org.scala-native" % "test-interface_native0.3_2.11" % "0.3.1"
```

Comment:
see https://github.com/scala-native/sbt-crossproject#cross-compiling-scalajs-jvm-and-native